File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for (int i=0;i < nums.size();i++){
            for(int j=0;j< nums.size();j++){
                if(nums[i]+ nums[j] == target && i!=j && i<j){
                    std::vector<int> list;
                    list.push_back(i);
                    list.push_back(j);
                    return list;
                }
                else if(nums[i]+ nums[j] == target && i!=j && j<i){
                    std::vector<int> list;
                    list.push_back(j);
                    list.push_back(i);
                    return list;
                }
            }
        }
    }
};

File: Data Structures & Algorithms/combination-target-sum/submission-0.py
class Solution:
    def combinationSum(self, nums: List[int], target: int) -> List[List[int]]:
        answer, curset = [], []
        self.helper(0,nums, answer, curset, target)
        return answer


    def helper(self,i, nums, answer, curset, target):
        if sum(curset) == target:
            answer.append(curset.copy())
            return 

        if sum(curset) > target:
            return

        for j in range(i, len(nums)):
            curset.append(nums[j])
            self.helper(j, nums, answer, curset, target)
            curset.pop()